### PR TITLE
Logo Added #61

### LINKED ISSUE #61: Add logo to website
Adding finalized logo to woodog website.

### PR DESCRIPTION
**Pull request by ishitakapoor26**
Fixes #61 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

---

### CODE REVIEW

**Comment by ishitakapoor26** [comment]
@hariramjp777 pls review the pr.

**Comment by hariramjp777** [comment]
> @hariramjp777 pls review the pr.

Yeah, I'm checking your PR.

File: Woodog/templates/partials/base.html [hunk 1]
```diff
@@ -118,7 +118,7 @@
 
 <nav id="header" class="navbar navbar-expand-lg">
   <div class="container-fluid">
-    <a class="navbar-brand logo" href="#"><img src="{% static 'img/woodog-logo.svg' %}"></a>
+    <a class="navbar-brand logo" href="#"><img src="{% static 'Woodog\Woodog\static\img\woodog-logo1.svg' %}"></a>
```
Comment (hariramjp777) [inline Woodog/templates/partials/base.html:121]: It should be,
``

**Review by hariramjp777 — CHANGES_REQUESTED** [review]
@ishitakapoor26 changes requested :+1:

File: Woodog/templates/partials/base.html [hunk 2]
```diff
@@ -118,7 +118,7 @@
 
 <nav id="header" class="navbar navbar-expand-lg">
   <div class="container-fluid">
-    <a class="navbar-brand logo" href="#"><img src="{% static 'Woodog\Woodog\static\img\woodog-logo1.svg' %}"></a>
+    <a class="navbar-brand logo" href="#"><img src="{% static 'Woodog\Woodog\static\img\woodog-logo1.png' %}"></a>
```
Comment (hariramjp777) [inline Woodog/templates/partials/base.html]: Please change this line to,
```html
<img src="{% static 'img/woodog-logo1.png' %}">
```

**Comment by hariramjp777** [comment]
Since you've used force push, I couldn't check your PR. I'm closing this issue. Please make a new PR.

**Pull request closed without merging**